code/game/objects/structures/mineral_doors.dm

```diff
@@ -83,26 +83,22 @@
 		icon_state = "[base_icon_state][smoothing_flags ? "-[smoothing_junction]" : ""]"
 
 
-/obj/structure/mineral_door/attackby(obj/item/W, mob/living/user)
+/obj/structure/mineral_door/attackby(obj/item/attacking_item, mob/living/user)
```

Comment:
Given that this is explicitly a pr to change resin doors, why are the changes on mineral doors as a whole?